roles/sro_capture_deployment_state/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+- name: Get all the nodes
+  failed_when: false
+  shell:
+    oc get nodes
+       > {{ artifact_extra_logs_dir }}/all_nodes.list
+
+- name: Get all the machines
+  failed_when: false
+  shell:
+    oc get machines -A
+       > {{ artifact_extra_logs_dir }}/all_machines.list
+
+- name: Get the deployments in openshift-special-resource-operator
+  failed_when: false
+  shell:
+    oc get deployments
+       -n openshift-special-resource-operator
+       > {{ artifact_extra_logs_dir }}/all_sro_deployments.list
+
+- name: Get the Pods in openshift-special-resource-operator
+  failed_when: false
+  shell: 
+    oc get pods
+       -n openshift-special-resource-operator
+       -owide
+       > {{ artifact_extra_logs_dir }}/all_sro_pods.list
+
+- name: Get the sro Pod object yaml
+  shell: |
+    dest_file="{{ artifact_extra_logs_dir }}/sro_operator_pod.yml";
+    oc get pods -n openshift-special-resource-operator -l control-plane=controller-manager -o yaml >> $dest_file;
```

Comment:
why is this `shell` not written as the other ones? (I prefer the other ones above if I didn't miss anything)
and why using the `>>` redirection, there's nothing to add to, AFAICS